packages/terra-dropdown-button/src/_DropdownButtonBase.jsx

```diff
@@ -72,6 +72,7 @@ class DropdownButtonBase extends React.Component {
       { 'is-block': isBlock },
       { disabled },
       customProps.className,
+      'ignore-react-onclickoutside',
```

Comment:
Did you try directly passing the `disableOnClickOutside ` prop to the Hook.Content component? 
Ref: https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/_Dropdown.jsx#L79

Reply:
The problem with that is I still want to get notified for clicks outside the `DropdownButton` component.
https://github.com/cerner/terra-core/blob/8c1821643741909627549952b976d1c5f534a9ea/packages/terra-dropdown-button/src/_Dropdown.jsx#L44
`Select` uses `onBlur` to account for this which works if you don't want to know if what got clicked on to move focus was a `button`.